pandas/core/index.py

```diff
@@ -1146,7 +1163,7 @@ def _ensure_compat_concat(indexes):
 
         return indexes
 
-    def take(self, indexer, axis=0):
+    def take(self, indexer, axis=0, **kwargs):
```

Comment:
Why is `**kwargs` here? Accepting arbitrary keyword arguments is a little dangerous because it doesn't catch errors.